djinni/jni/djinni_support.cpp

```diff
@@ -75,7 +75,7 @@ JNIEnv * jniGetThreadEnv() {
     jint get_res = g_cachedJVM->GetEnv(reinterpret_cast<void**>(&env), JNI_VERSION_1_6);
     #ifdef EXPERIMENTAL_AUTO_CPP_THREAD_ATTACH
     if (get_res == JNI_EDETACHED) {
-        get_res = g_cachedJVM->AttachCurrentThread(reinterpret_cast<void**>(&env), nullptr);
+        get_res = g_cachedJVM->AttachCurrentThread(&env, nullptr);
```

Comment:
```suggestion
#ifdef __ANDROID__
    get_res = g_cachedJVM->AttachCurrentThread(&env, nullptr);
#else
    get_res = g_cachedJVM->AttachCurrentThread(reinterpret_cast<void**>(&env), nullptr);
#endif
```

Reply:
this change is included in the PR to the src repo , see https://github.com/NetsoftHoldings/djinni-support-lib/pull/1